config/routes.rb

```diff
@@ -237,6 +237,8 @@
     end
   end
 
+  get "challenges/48in23/implementation_status", to: 'challenges#implementation_status'
```

Comment:
I struggled a bit with Rails routing, so I just hacked this together

Reply:
See comment below.